eclipse-collections-api/src/main/java/org/eclipse/collections/api/stack/ImmutableStack.java

```diff
@@ -52,6 +53,20 @@
 
     ImmutableStack<T> pop(int count);
 
+    /**
+     * @since 11.1
+     */
+    default Pair<T, ImmutableStack<T>> peekAndPop() {
```

Comment:
The code formatting is not following the project standard here. Beginning curly brace should be on next line.